pkg/export/gcm/export_gcm_test.go

```diff
@@ -0,0 +1,100 @@
+package gcm_test
+
+import (
+	"testing"
+	"time"
+
+	"github.com/GoogleCloudPlatform/prometheus-engine/pkg/export/gcm/promtest"
+	"github.com/prometheus/client_golang/prometheus"
+)
+
+func TestExport_CounterReset(t *testing.T) {
+	const interval = 30 * time.Second
+
+	prom := promtest.Prometheus("quay.io/prometheus/prometheus:v2.47.2")
```

Comment:
Since this method has a default Prometheus image, why not use that to ensure this test and future tests use the same version?

Reply:
Hm, I don't get you argument. Why not using default would make future tests use different version? 🤔 

I think I will remove default (YAGNI), we don't need implicit versions here.

Reply:
If it's a static version in each test, I fear they may go out of sync.